src/EmailFactory.php

```diff
@@ -29,8 +29,8 @@ public function create(string $mailbox, Horde_Imap_Client_Data_Fetch $hordeEmail
         // Parse the message body
         $parser = new Parser();
         $parser->setText($hordeEmail->getFullMsg());
-        $htmlContent = utf8_encode((string) $parser->getMessageBody('html'));
```

Comment:
@mnapoli Since the symptom is demonstrably a double-encoding of ISO-8859-1 -> UTF-8 and that is precisely what `utf8_encode()` would do here, I am very confident that this would fix the problem.

Just as a concrete example of this being what is happening:

https://externals.io/thread/787#email-14713

> Sara Golemon a Ã©crit :

[In ISO-8859-1](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#Codepage_layout):
```
'Ã' === "\xC3"
'©' === "\xA9"
```

[In UTF-8](http://www.fileformat.info/info/unicode/char/e9/index.htm):
```
'é' === "\xC3\xA9"
```